### DISCUSSION TITLE
How to set initial Balance for account with many imported transactions

### DISCUSSION THREAD

**Discussion by PiotrEsse** [firefly-iii/firefly-iii#6986] [General]
Good day, 
I am newbee. 
Ive  imported transactions from many years and now I would to set initial balance for account.

After importing transaction account balance differ from real bank account balance. Ive tried to set up Virtual balance and put a balance value but still nothing have changed. Only balance discrepancy columns have changed.
How to make that account balance in FireFly3 is same as Bank account balance?
Please help.

**Comment by JC5** [comment]
Change the opening balance, and set the date before the first imported transaction.

**Reply by PiotrEsse** [reply]
Thank You. It worked!
